profiler/src/main/java/com/navercorp/pinpoint/profiler/instrument/classloading/PlainClassLoaderHandler.java

```diff
@@ -107,6 +107,9 @@ private boolean isPluginPackage(String className) {
         return pluginConfig.getPluginPackageFilter().accept(className);
     }
 
+    private boolean isPluginPackageRequirementMeet(String className) {
```

Comment:
How about `meetsPluginPackageRequirement` instead of `is~` ?

Reply:
`satisfyingPackageRequirements` also

Reply:
thanks, I will use `meetsRequirement`